src/modules/dashboard/components/CreateColonyWizard/StepSelectToken.tsx

```diff
@@ -57,6 +59,14 @@ const MSG = defineMessages({
     defaultMessage:
       'Not a valid token. Only ERC20 tokens with 18 decimals are supported.',
   },
+  xdaiAddress: {
+    id: 'dashboard.CreateColonyWizard.StepSelectToken.xdaiAddress',
+    defaultMessage: 'You cannot use XDAI token as a native token for colony.',
+  },
+  ethAddress: {
+    id: 'dashboard.CreateColonyWizard.StepSelectToken.xdaiAddress',
+    defaultMessage: 'You cannot use ETH token as a native token for colony.',
+  },
```

Comment:
This works as well, but when I suggested you use the `DEFAULT_NETWORK_TOKEN` constant, I was expecting you to pass it along as a value.

```suggestion
  ethAddress: {
    id: 'dashboard.CreateColonyWizard.StepSelectToken.xdaiAddress',
    defaultMessage: 'You cannot use {tokenSymbol} token as a native token for colony.',
  },
```

Then call it passing along a values object:
```
values={{ tokenSymbol: DEFAULT_NETWORK_TOKEN.symbol }}
```

That way you don't really care what network we are on, you just care that the token address is not `0x00000`

Reply:
See an usage example here:

https://github.com/JoinColony/colonyDapp/blob/c3299c8db8a1f2a40bd18deeb4a51495893d81c7/src/modules/dashboard/components/CreateColonyWizard/StepUserName.tsx#L155-L161